### PR TITLE
[partition] keep 'option' in properties

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

In the last PR, 'option' keyword was not removed until the control is passed to model.py. Looking up every keyword with option keyword. 
This should also fix the actions failure. Tested it manually.

### CODE REVIEW

**Review by lanking520 — APPROVED** [review]

File: serving/docker/partition/partition.py [hunk 1]
```diff
@@ -99,20 +99,20 @@ def set_environmental_vars(self):
         if os.environ.get("PYTHONPATH"):
             python_path.append(os.environ.get("PYTHONPATH"))
         python_path.append(PYTHON_CACHE_DIR)
-        if 'model_dir' in self.properties:
-            python_path.append(self.properties['model_dir'])
+        if 'option.model_dir' in self.properties:
```
Comment (frankfliu) [inline serving/docker/partition/partition.py:102]: 1. Not all perperties has `option.` prefix. 
2. We don't think we are using `model_dir` in the `serving.perperties`, model_dir is passed to `model.py` ad current directory

File: serving/docker/partition/partition.py [hunk 2]
```diff
@@ -121,23 +121,25 @@ def download_config_from_hf(self):
...
             config_files += glob.glob(os.path.join(model_dir, pattern))
 
         for file in config_files:
-            shutil.copy(file, dst=self.properties['save_mp_checkpoint_path'])
+            shutil.copy(file,
+                        dst=self.properties['option.save_mp_checkpoint_path'])
 
     def upload_checkpoints_to_s3(self):
-        if 'upload_checkpoints_s3url' not in self.properties:
+        if 'option.upload_checkpoints_s3url' not in self.properties:
```
Comment (frankfliu) [inline serving/docker/partition/partition.py:137]: No need to add 'option.` to intermediate key using only for partition

Comment (sindhuvahinis) [reply serving/docker/partition/partition.py]: Okay, fixed

File: serving/docker/partition/partition.py (hunk 1 above)
Comment (sindhuvahinis) [reply serving/docker/partition/partition.py]: Alright, removed option for model_dir

**Review by frankfliu — APPROVED** [review]

**Pull request merged**